packages/datetime/src/dateInput.tsx

```diff
@@ -98,6 +98,12 @@ export interface IDateInputProps extends IDatePickerBaseProps, IProps {
     outOfRangeMessage?: string;
 
     /**
+     * Whether the date popover should be inline.
+     * @default true
+     */
+    popoverInline?: boolean;
```

Comment:
Hmmm I setup circle as in the docs... I wonder if it wasn't happy that I pushed this before having circle set up. I may need to make a new PR to fix.

@giladgray Is there a timeframe for that investigation? I have a PR ready that is blocked on this functionality that is preferable to get in this week. 

As for focus issues, I played around with it not being inline and the only focus issues I noticed came from putting the Date Input into a popover, where clicking on the Date popover would cause both to close. My use case is not in a popover, so it should be fine.